pkg/apis/app/v2/zz_generated.deepcopy.go

```diff
@@ -1,6 +1,6 @@
 // +build !ignore_autogenerated
 
-// Code generated by deepcopy-gen. DO NOT EDIT.
+// Code generated by operator-sdk. DO NOT EDIT.
```

Comment:
This appears when using newer sdk version. Can you please ensure version 0.7.0 was used and fix commit if necessary?